integration-tests/src/test/groovy/com/okta/cli/test/CommandRunner.groovy

```diff
@@ -104,6 +104,7 @@ class CommandRunner {
         }
 
         process.waitForOrKill(timeout().toMillis())
+        process.closeStreams()
```

Comment:
I _think_ this will fix some of the flakey tests (though i would have expected the previous line to do this for me)